python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -576,7 +625,12 @@ def get_self_rank(self) -> Optional[int]:
         def get_actor_handles(self) -> List["ray.actor.ActorHandle"]:
             return self._actor_handles
 
-        def send(self, value: "torch.Tensor", peer_rank: int) -> None:
+        def send(
+            self,
+            future: "ray.dag.dag_operation_future.DAGOperationFuture['torch.Tensor']",
```

Comment:
Why does this need to take in a future instead of the torch.Tensor value as before?

Reply:
This was not updated yet. Done now.